src/ameedf/jb8223/assignments/RotationCheck.java

```diff
@@ -42,6 +44,92 @@
  */
 public class RotationCheck {
     public static void main(String[] args) {
-        // TODO: enter your code here :)
+		Scanner in = new Scanner(System.in);
+		int firstNumber = 0, secondNumber = 0;
+
+		//Get input from the user.
+		do {
+			System.out.print("Enter firstNumber [0 to 1,000,000,000]:");
+			firstNumber = in.nextInt();
+		} while (firstNumber < 0 || firstNumber > 1000000000);
+
+		do {
+			System.out.print("Enter secondNumber [0 to 1,000,000,000 and not equal to "+firstNumber+" ]:");
+			secondNumber = in.nextInt();
+		} while (secondNumber < 0 || secondNumber > 1000000000 || firstNumber == secondNumber);
+		
+		in.close();
+		
+		//Calculate the length of the numbers
+		int firstNumberLenght = 0, tempfirstNumber = firstNumber;
```

Comment:
leng**th**  and not leng**ht**
tempfirstNumber - camel case is temp**F**irstNumber

Reply:
Yes,  typo, fixed.